docs/minio-client-complete-guide.md

```diff
@@ -531,8 +531,8 @@ FLAGS:
   --incomplete, -I              remove incomplete uploads
   --fake                        perform a fake remove operation
   --stdin                       read object names from STDIN
-  --older-than value            remove objects older than N days (default: 0)
-  --newer-than value            remove objects newer than N days (default: 0)
+  --older-than value            Remove objects older than L days, M hours and N minutes LNM[d|h|m]. (default: 0)
```

Comment:
`Remove` should be `remove`. Rest of the text starts with a lower case letters.

Reply:
Done.